### ISSUE TITLE
Can't get the integration to start since HA 2023.7.2

### ISSUE THREAD

**Issue by yannpub** [saniho/apiNetatmo#7]
I have lost access to the integration since couple of days (not sure it is 100% related to the core upgrade, but happened the overall approximative timeframe). Here is the log from my latest restart of HA:

Logger: homeassistant.components.sensor
Source: custom_components/apiNetatmo/sensor.py:98
Integration: Sensor (documentation, issues)
First occurred: July 16, 2023 at 11:52:25 PM (1 occurrences)
Last logged: July 16, 2023 at 11:52:25 PM

Error while setting up apiNetatmo platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 353, in _async_setup_platform
    await asyncio.shield(task)
  File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/apiNetatmo/sensor.py", line 98, in setup_platform
    for myStationKeys in lstStations.keys():
                         ^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'keys'

**Comment by jsapede** [comment]
same here

**Comment by saniho** [comment]
the error is related to the new version of the netatmo api
the pre release 1.1.0.0 used this new one...please test it
Warning : change in the yaml sensor config needed

**Comment by yannpub** [comment]
Edit: I managed to restart HA, looks like I did not restart it properly, and was still using the old api in cache. Now HA restartrs, but I can't get the sensors to be created. Did I create the proper refresh_token by selecting 'Read_Station' ? 

Edit/Edit: After couple of changes/restarts, I got my sensors back ! All good now !

The YAML config:
sensor:
- platform: apiNetatmo
  host: 70:ee:50:3e:e2:5c
  refresh_token: <REFRESH_TOKEN GENERATED AFTER READ_STATION>
  code: <CLIENT_ID>
  token: <CLIENT_SECRET>
  scan_interval: 600

Thanks!
